book/src/concepts/event-channel.md

```diff
@@ -60,6 +60,9 @@ Then, when you want to read the events:
 Note that you only need to have a read access to the channel when reading events.
 It is the `ReaderId` that needs to be mutable to keep track of where your last read was.
 
+**IMPORTANT: The event channel automatically grows as events are added to it and shrinks only to a point where all readers have been through.
```

Comment:
`and shrinks only to a point where all readers have been through.` I would change this to `and only decreases in size once all readers have read through the older events.`